build.gradle

```diff
@@ -62,6 +62,8 @@ tasks.withType(JavaCompile) {
 }
 tasks.withType(Test) {
     systemProperty "file.encoding", "UTF-8"
+    jvmArgs("--add-opens", "java.base/java.time=ALL-UNNAMED")
```

Comment:
Add these two lines as mock will throw exception? Xun bumpted `mockito` version to fix some mock issue in this PR https://github.com/opensearch-project/ml-commons/pull/248. @Zhangxunmt you can add more details if that's the same issue.

Reply:
Integ tests are passing .If the mocks had issue it won't right ?

Reply:
I see the CI passed. I mean if don't add these two lines, the test will fail?

Reply:
Talked with @aksingh-es  offline, will throw these errors if remove line 65,66
```
ModelManagerTests > getThresholdingResult_returnExpectedToListener FAILED
  java.lang.reflect.InaccessibleObjectException: Unable to make private void java.time.Duration.readObject(java.io.ObjectInputStream) throws java.io.InvalidObjectException accessible: module java.base does not "opens java.time" to unnamed module @20fa23c1
    at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:354)
    at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:297)
    at java.base/java.lang.reflect.Method.checkCanSetAccessible(Method.java:199)
    at java.base/java.lang.reflect.Method.setAccessible(Method.java:193)
    at org.powermock.reflect.internal.WhiteboxImpl.doGetAllMethods(WhiteboxImpl.java:1508)
    at org.powermock.reflect.internal.WhiteboxImpl.getAllMethods(WhiteboxImpl.java:1482)
    at org.powermock.reflect.internal.WhiteboxImpl.getMethods(WhiteboxImpl.java:1750)
    at org.powermock.reflect.internal.WhiteboxImpl.getMethods(WhiteboxImpl.java:1789)
    at org.powermock.reflect.internal.WhiteboxImpl.getBestMethodCandidate(WhiteboxImpl.java:1008)
    at org.powermock.core.MockInvocation.findMethodToInvoke(MockInvocation.java:58)
    at org.powermock.core.MockInvocation.init(MockInvocation.java:35)
    at org.powermock.core.MockInvocation.<init>(MockInvocation.java:22)
    at org.powermock.core.MockGateway.doMethodCall(MockGateway.java:155)
    at org.powermock.core.MockGateway.methodCall(MockGateway.java:138)
    at org.opensearch.ad.ml.ModelManagerTests.setup(ModelManagerTests.java:190)
```

```
ModelManagerTests > getPreviewResults_returnNoAnomalies_forNoAnomalies FAILED
  java.lang.reflect.InaccessibleObjectException: Unable to make public void java.util.stream.ReferencePipeline$Head.forEach(java.util.function.Consumer) accessible: module java.base does not "opens java.util.stream" to unnamed module @20fa23c1
    at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:354)
    at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:297)
    at java.base/java.lang.reflect.Method.checkCanSetAccessible(Method.java:199)
    at java.base/java.lang.reflect.Method.setAccessible(Method.java:193)
    at org.powermock.reflect.internal.WhiteboxImpl.doGetAllMethods(WhiteboxImpl.java:1508)
    at org.powermock.reflect.internal.WhiteboxImpl.getAllMethods(WhiteboxImpl.java:1482)
    at org.powermock.reflect.internal.WhiteboxImpl.getMethods(WhiteboxImpl.java:1750)
    at org.powermock.reflect.internal.WhiteboxImpl.getMethods(WhiteboxImpl.java:1789)
    at org.powermock.reflect.internal.WhiteboxImpl.getBestMethodCandidate(WhiteboxImpl.java:1008)
    at org.powermock.core.MockInvocation.findMethodToInvoke(MockInvocation.java:58)
    at org.powermock.core.MockInvocation.init(MockInvocation.java:35)
    at org.powermock.core.MockInvocation.<init>(MockInvocation.java:22)
    at org.powermock.core.MockGateway.doMethodCall(MockGateway.java:155)
    at org.powermock.core.MockGateway.methodCall(MockGateway.java:138)
    at org.opensearch.ad.ml.ModelManagerTests.getPreviewResults_returnNoAnomalies_forNoAnomalies(ModelManagerTests.java:856)
```